app/controllers/admin/base_admin_controller.rb

```diff
@@ -31,7 +31,10 @@ def end_expired_admin_sessions
     end
 
     def ensure_authenticated_user
-      redirect_to admin_sign_in_path unless admin_signed_in?
+      unless admin_signed_in?
+        session[:user_signed_out_path] = request.fullpath
```

Comment:
Without reading the code further I was a bit confused by the naming of this. I thought it was going to be something we would redirect to when signing out. Maybe something like `requested_admin_path` would be more intention revealing?

Reply:
Yeah, this confused me too. I agree with Robbie, `requested_admin_path` would work better I think 👍